framework/src/application/application.js

```diff
@@ -477,10 +477,12 @@ class Application {
 						this._node.actions.updateForgingStatus(action),
 				},
 				getForgingStatusOfAllDelegates: {
+					// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
we dont need them,  `this._node.actions.getForgingStatusOfAllDelegates` is sync function, so you can remove `async`  from handler `handler: () =>`